returnn/tf/network.py

```diff
@@ -3576,7 +3580,7 @@ def assign_var(self, var, session):
         return
       self.assigned = True
       values_dict = {
-        name: self.reader.get_tensor(self.prefix_param_name + name)
+        name: (self.reader.get_tensor(self.var_name_mapping[name]) if name in self.var_name_mapping else self.reader.get_tensor(self.prefix_param_name + name) )
```

Comment:
```suggestion
        name: self.reader.get_tensor(self.var_name_mapping.get(name, self.prefix_param_name + name))
```
Is easier, shorter, and less redundant (not two separate `get_tensor` calls in the code).